generators/server/templates/pom.xml.ejs

```diff
@@ -82,10 +82,10 @@
         <profile.tls />
 
         <!-- Dependency versions -->
-        <jhipster-dependencies.version>3.7.0-SNAPSHOT</jhipster-dependencies.version>
+        <jhipster-dependencies.version><%= JHIPSTER_DEPENDENCIES_VERSION %></jhipster-dependencies.version>
         <!-- The spring-boot version should match the one managed by
```

Comment:
Just an idea while I see that: would it be possible to add a conditional on this property?
if (JHIPSTER_DEPENDENCIES_VERSION .endwith("-SNAPSHOT") {
// declare Snapshot repository in pom or gradle build file
}

Reply:
This is what I want to setup for SpringBoot in the `spring-boot-2.3` branch, by conditionally defining Spring milestones repository. 
However I don't see how it will works for JHipster dependencies, since the snapshots are not regularly published and thus are not really useful (that might change if we use GitHub Packages in the future).

Reply:
Automated publication of jhipster-dependencies snapshots is somewhat that is easily doable :-)

Reply:
Yep. I'm working on it :)

Reply:
@Tcharl : Done! Julien was in fact already publishing automatically snapshots, so I've added conditionally the snapshot repository ;)